### ISSUE TITLE
31 pages of unusable skins

### ISSUE THREAD

**Issue by USER120509** [8fn/LegendarySkins-PD2#9]
Since a few months now I got 31 pages of unusable skins for all weapons before the actual skins for the specific weapon shows up. I saw that you pushed an update to fix this problem, but sadly it still persists for me. I have to say that I was playing on Epic Games, but transferred my save file to Steam, but nonetheless I still have that issue

**Comment by VictorKruger** [comment]
Have you tried running the steam version of the game with the mod enabled/installed?

**Comment by USER120509** [comment]
Yes, the mod is installed and enabled. Even with the mod disabled I still have 31 pages of the unusable skins unfortunately
